config/env_constants.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+
+# Define the application directory
+BASE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
```

Comment:
One notable difference here is the change to BASE_DIR to keep it the same, otherwise we'd end up with `session.db`, etc. in `/config/`.